codespell_lib/data/dictionary.txt

```diff
@@ -9664,6 +9667,8 @@ intelectual->intellectual
 intelegence->intelligence
 intelegent->intelligent
 intelegently->intelligently
+inteligable->intelligible, intelligibility,
```

Comment:
Sorry, I meant a separate suggestion of inteligability->intelligibility

Reply:
Done